src/main/webapp/dashboard-licenses.xhtml

```diff
@@ -61,10 +62,46 @@
                             <p:ajax update="@this" listener="#{LicenseListingPage.saveLicenseActiveStatus(license)}"/>
                         </p:selectBooleanCheckbox>
                     </p:column>
+                    <p:column headerText="#{bundle['dashboard.license.action']}"
+                              style="width: 10%; text-align: center;">
+                        <p:commandLink type="button"
+                                       styleClass="btn btn-default bootstrap-button-tooltip"
+                                       title="#{bundle['dashboard.license.localePreviewButton']}"
+                                       update=":licenseLocalizedPreviewForm"
+                                       action="#{LicenseListingPage.setSelectedLicense(license)}"
+                                       onclick="PF('licenseLocalizedPreview').show();">
+                            <span style="text-align: center" class="glyphicon glyphicon-eye-open"/>
+                        </p:commandLink>
+                    </p:column>
                 </p:dataTable>
 
             </h:form>
 
+            <p:dialog modal="true" widgetVar="licenseLocalizedPreview" id="licenseLocalizedPreview"
+                      styleClass="disable-scroll" width="600"
+                      draggable="false" closable="false" resizable="false">
+                <h:form id="licenseLocalizedPreviewForm">
```

Comment:
Actually it is, the view is built when you are entering the page and at this time 'license' is null, it is set later, you have to update the dialog so it will show updated values.